packages/design-system/src/components/primitives/numeric-gesture-control.stories.tsx

```diff
@@ -18,7 +18,7 @@ const useNumericScrubControl = ({
     if (ref.current === null) return;
     ref.current.value = String(value);
     const { disconnectedCallback } = numericScrubControl(ref.current, {
-      initialValue: value,
+      getValue: () => value,
```

Comment:
do you intent to call it any time or only as an initial value? if only as initial, then we couldn't call it getInitialValue